src/Symfony/Component/Security/Http/Firewall/BasicAuthenticationListener.php

```diff
@@ -80,7 +80,7 @@ public function handle(GetResponseEvent $event)
             }
 
             if (null !== $this->logger) {
-                $this->logger->info(sprintf('Authentication request failed for user "%s": %s', $username, $failed->getMessage()));
+                $this->logger->info(sprintf('Basic authentication failed for user "%s"', $username), array('exception' => $failed));
```

Comment:
I'm not sure if I can do this for everything, I'll see what I can do as most cases _can_ be changed.

Regarding the messages, should log messages end with a `.`? It's mixed up right now and I'd like to fix this while at it.